src/module-elasticsuite-catalog/view/adminhtml/web/template/form/element/product-sorter.html

```diff
@@ -17,6 +17,11 @@
     </div>
     
     <div data-bind="if: hasProducts()">
+
+        <p>
+            <input type="text" name="search" id="elasticsuite-preview-search" class="admin__control-text"/>
+            <a href="#" data-bind="click: searchProducts, text: messages.search" class="action-default"></a>
```

Comment:
Should trigger when ENTER is hit without needing to click : maybe the button can be removed, and replace by a "reset" button in case a search is active.

Reply:
I added this. The search button is still there but a reset link is added.